omnibus_overrides.rb

```diff
@@ -20,7 +20,7 @@
 override "nokogiri", version: "1.13.1"
 override "openssl", version: mac_os_x? ? "1.1.1m" : "1.0.2zb"
 override "pkg-config-lite", version: "0.28-1"
-override "ruby", version: "3.0.3"
+override "ruby", version: aix? ? "3.0.3" : "3.1.2"
```

Comment:
This seems outside the scope of what this PR is intended to so - it will change the omnibus build to use ruby 3.1, but I don't think all of our gems are compatible yet.

Reply:
undone, good catch.